### ISSUE TITLE
Cant get program to run

### ISSUE THREAD

**Issue by tubetime3** [matt1484/bl3_auto_vip#22]
After I enter the email and password nothing happens

**Comment by matt1484** [comment]
What operating system and version?

**Comment by tubetime3** [comment]
Windows 10 version 1903

**Comment by matt1484** [comment]
I meant the app version, sorry. The latest version is 2.1, do you have that downloaded?

**Comment by tubetime3** [comment]
Yes I believe so I downloaded the windows ZIP file located here https://github.com/matt1484/bl3_auto_vip/releases/tag/v2.1

**Comment by remiX-** [comment]
Same thing happens to me, I downloaded the .exe straight. Windows 10 1903

**Comment by Zaf9670** [comment]
I believe I'm having a similar issue. The utility opens, I put in the username and password. It quickly says something along the lines of successfully logged in and then terminates itself.

**Comment by Zaf9670** [comment]
> 
> 
> I believe I'm having a similar issue. The utility opens, I put in the username and password. It quickly says something along the lines of successfully logged in and then terminates itself.

Turns out I was using my 2K password instead of my Gearbox password. It logged in successfully after that.

**Comment by matt1484** [comment]
Based on reports from other users, it seems that more likely than not, the password/account is not correctly setup. Only other thing I can think of is firewall blocking connections somehow.
